### PR TITLE
add ansible script to create a Win VM

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this?**

/kind tests

**What does this PR do / why we need it**:
This PR will add ansible script to create VM.

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [x] I have read the [test guidelines](https://github.com/redhat-developer/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-150.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-150.html)

**Comment by feloy** [comment]
OpenShift Tests finished with errors.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-179.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-179.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-163.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-163.html)

File: scripts/ansible/kubernetes-cluster/create-vm.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    - name: Configure VSI
+      ibm_is_instance:
+        name: "{{ name_prefix }}-windows-vsi"
+        state: available
+        id: "{{ vsi.id | default(omit) }}"
+        vpc: "{{ vpc.id }}"
+        profile: "{{ vsi_profile }}"
+        image: "{{ image_dict[vsi_image] }}"
+        keys:
+          - "r010-0f01beeb-9b45-4406-93bb-4a8604c8a3e0"         #sshkey id ansible-ssh-key in developer qe group in VPC sshkey
```
Comment (feloy) [inline scripts/ansible/kubernetes-cluster/create-vm.yml:70]: Please use a variable for this key ID

Comment (anandrkskd) [reply scripts/ansible/kubernetes-cluster/create-vm.yml]: Understood added!

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-172.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-172.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-185.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-185.html)

**Comment by feloy** [comment]
OpenShift Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-201.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-201.html)

File: scripts/ansible/kubernetes-cluster/create-vm.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - name: Configure VSI
+      ibm_is_instance:
+        name: "{{ name_prefix }}-windows-vsi"
+        state: available
+        id: "{{ vsi.id | default(omit) }}"
+        vpc: "{{ vpc.id }}"
+        profile: "{{ vsi_profile }}"
+        image: "{{ image_dict[vsi_image] }}"
+        keys:
+          - " {{ sshkey_id }} "         #sshkey id ansible-ssh-key in developer qe group in VPC sshkey
```
Comment (feloy) [inline scripts/ansible/kubernetes-cluster/create-vm.yml:70]: Please also move the comment to the line defining the specific key ID

Comment (anandrkskd) [reply scripts/ansible/kubernetes-cluster/create-vm.yml]: Done!
Thanks **:)**

**Comment by feloy** [comment]
OpenShift Tests finished with errors.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-225.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-openshift-tests-225.html)

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-195.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-kubernetes-tests-195.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-208.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5288-unit-tests-208.html)

**Pull request merged**